tests/src/rules/no-duplicates.js

```diff
@@ -119,49 +119,48 @@ ruleTester.run('no-duplicates', rule, {
 
     test({
       code: "import { x, /* x */ } from './foo'; import {//y\ny//y2\n} from './foo'",
-      output: "import { x, /* x */ //y\ny//y2\n} from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import {x} from './foo'; import {} from './foo'",
-      output: "import {x} from './foo'; ",
+      output: "import { x } from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import {x} from './foo'; import {} from './foo'; import {/*c*/} from './foo'; import {y} from './foo'",
-      output: "import {x/*c*/,y} from './foo';   ",
+      output: "import { x, y } from './foo';  import {/*c*/} from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import { } from './foo'; import {x} from './foo'",
-      output: "import { x} from './foo'; ",
+      output: "import { x } from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import './foo'; import {x} from './foo'",
-      output: "import {x} from './foo'; ",
+      output: "import { x } from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import'./foo'; import {x} from './foo'",
-      output: "import {x} from'./foo'; ",
+      output: "import { x } from './foo'; ",
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
     test({
       code: "import './foo'; import { /*x*/} from './foo'; import {//y\n} from './foo'; import {z} from './foo'",
-      output: "import { /*x*///y\nz} from './foo';   ",
+      output: "import { z } from './foo'; import { /*x*/} from './foo'; import {//y\n} from './foo'; ",
```

Comment:
this isn't much of an improvement; it ends up leaving behind two useless import statements just for the comments.